core/templates/dev/head/domain/exploration/EditableExplorationBackendApiServiceSpec.js

```diff
@@ -202,35 +204,38 @@ describe('Editable exploration backend API service', function() {
       var successHandler = jasmine.createSpy('success');
       var failHandler = jasmine.createSpy('fail');
 
-      $httpBackend.expect('GET', '/createhandler/data/0').respond(
-        sampleDataResults);
+      $httpBackend.expect('GET', '/createhandler/data/0')
+        .respond(sampleDataResults);
 
-      EditableExplorationBackendApiService.fetchExploration('0').then(
-        function(data) {
+      EditableExplorationBackendApiService.fetchExploration('0')
+        .then(function(data) {
           exploration = data;
         });
       $httpBackend.flush();
 
       exploration.title = 'New Title';
       exploration.version = '2';
 
-      $httpBackend.expect('PUT', '/createhandler/data/0').respond(
-        exploration);
+      $httpBackend.expect('PUT', '/createhandler/data/0')
+        .respond(exploration);
 
       // Send a request to update exploration
-      EditableExplorationBackendApiService.updateExploration(
-        exploration.exploration_id, exploration.version,
-        'Minor edits', []).then(
-        successHandler, failHandler);
+      EditableExplorationBackendApiService
+        .updateExploration(
+          exploration.exploration_id,
+          exploration.version,
+          'Minor edits', [])
+        .then(successHandler, failHandler);
```

Comment:
move ) from previous line to start of this line

Reply:
Done.